### DISCUSSION TITLE
False Triggers on Filament Sensr in Klipper

### DISCUSSION THREAD

**Discussion by MACCAMENZIES** [xblax/flashforge_ad5m_klipper_mod#118] [Q&A]
Hi Everyone
Is anyone getting false triggers on the filament sensor?. Couple of times now have been running a print and will get a random pause for filament trigger. Filament is loaded no problems . I have turned off the filament sensor on the mainsail dashboard to temporality solve the issue. Is there a sensitivity setting ?

**Comment by THEGxHOxST** [comment]
You could try increasing the "event_delay: 1.0" to "event_delay: 3.0" in printer.base.cfg under "[filament_switch_sensor runout_sensor]" to maybe filter out false positives.

If that does not work, I hate to say it but check the wiring... 
I haven't had to or needed to so I have no way to tell you how to do this.

**Reply by MACCAMENZIES** [reply]
Thanks for that. I think the timing is the way to go as the sensor is working under the normal firmware. Ill try your suggestion on next print

**Comment by MACCAMENZIES** [comment]
No luck . Still pausing with filament loaded.
Working fine in the standard firmware like i said. Weird!

> You could try increasing the "event_delay: 1.0" to "event_delay: 3.0" in printer.base.cfg under "[filament_switch_sensor runout_sensor]" to maybe filter out false positives.
> 
> If that does not work, I hate to say it but check the wiring... I haven't had to or needed to so I have no way to tell you how to do this.

**Reply by THEGxHOxST** [reply]
For the time being you could add "pause_on_runout: False" so at least you'll be notified if it runs out and change  "event_delay: 1.0" back

**Comment by cmenard** [comment]
> Hi Everyone Is anyone getting false triggers on the filament sensor?. Couple of times now have been running a print and will get a random pause for filament trigger. Filament is loaded no problems . I have turned off the filament sensor on the mainsail dashboard to temporality solve the issue. Is there a sensitivity setting ?

I had this happen with version 003 during a couple of prints.  After updating to 004 I haven't had the issue again.

**Reply by MACCAMENZIES** [reply]
I changed the event delay all the way up to 6 just to see if it had any effect. It seems better but eventually did trigger the run out. Very strange.

**Comment by xblax** [comment]
> No luck . Still pausing with filament loaded.
> Working fine in the standard firmware like i said. Weird!

I didn't observe that yet with the mod. Btw. the standard firmware has the runout sensor disabled by default, maybe because it's not that reliable.

The sensor is very simple in design. That is just a steel ball that is pushed out of the way by the filament and then the steel ball triggers a micro switch.

**Reply by MACCAMENZIES** [reply]
Yes you could be right. The ball design is not the best. I liked the Prusa design better . Was more reliable for sure. Might look at pulling the switch out and see if can retro fit a different one. IF it had been filing on the stock firmware I would understand but then I got to thinking it may have been a voltage on pins thing but negated that due to the mechanical design with the ball
